src/cargo/core/features.rs

```diff
@@ -943,6 +1004,67 @@ impl GitoxideFeatures {
             internal_use_git2: false,
         }
     }
+
+    fn expecting() -> String {
+        let fields = vec!["`fetch`", "`checkout`", "`internal-use-git2`"];
+        format!(
+            "unstable 'gitoxide' only takes {} as valid inputs, for shallow fetches see `shallow-index,shallow-deps`",
```

Comment:
```suggestion
            "unstable 'gitoxide' only takes {} as valid inputs, for shallow fetches see `-Zgit=shallow-index,shallow-deps`",
```

Reply:
Shouldn't it be `-Zgitoxide`

Reply:
Looks like the old tip was wrong, too. See https://doc.rust-lang.org/cargo/reference/unstable.html, let me  correct it to `-Zgitoxide` also

Reply:
Sorry, I misunderstand